src/core/session/session.cpp

```diff
@@ -87,8 +89,16 @@ void Session::load()
         auto tabObj = tab.toObject();
         auto state = new TabState(this);
         state->setUrl(tabObj["url"].toString());
+        state->setTitle(tabObj["title"].toString());
+        state->setIcon(tabObj["icon"].toString());
         m_tabs.append(state);
     }
+
+    m_activeTab = std::max(m_tabs.count() - 1, 0);
+    if (root.find("activeTab") != root.end())
+    {
```

Comment:
Let's keep the opening curly brace in the same line for consistency-

Reply:
sorry for that braces, my old habit from c# :)

Reply:
No problem, thanks! :)